client/finality-grandpa/rpc/src/lib.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! RPC API for GRANDPA.
+
+#![deny(missing_docs)]
```

Comment:
```suggestion
#![warn(missing_docs)]
```

Please don't use `deny`, we've been through this couple of times already and it makes more harm than good.
Namely:
1. Can break the build out of the blue sometimes when there are changes in code generated by other libs or compiler changes.
2. Makes it hard to develop locally.

Reply:
We should still forbid missing documentation on CI, though.

Reply:
Yep all good points I agree with!